### DISCUSSION TITLE
Can't create new repository

### DISCUSSION THREAD

**Discussion by welovelain** [community/community#21650] [New to GitHub]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/a/ca2433006cc691b8d60bfbc2e291bd9110155702.png" data-download-href="/uploads/short-url/sQe0lbVc3gXqd3cC2U23u31f5Mm.png?dl=1" title="snapshot-22_04_2021_19-02-12" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/15702174/181066003-b8a39ff3-5b84-49b2-b12b-40f07411ca97.png" alt="snapshot-22_04_2021_19-02-12" data-base62-sha1="sQe0lbVc3gXqd3cC2U23u31f5Mm" width="672" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/a/ca2433006cc691b8d60bfbc2e291bd9110155702_2_672x500.png, https://aws1.discourse-cdn.com/github/original/3X/c/a/ca2433006cc691b8d60bfbc2e291bd9110155702.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/c/a/ca2433006cc691b8d60bfbc2e291bd9110155702.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/a/ca2433006cc691b8d60bfbc2e291bd9110155702_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">snapshot-22_04_2021_19-02-12</span><span class="informations">929×691 361 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Browser: FireFox<br>
Tried to disable UBlock, tried to clear cookies.</p>
<p>Tried to create public/private with any name.<br>
All results in this 500 error.</p>

**Comment by hasancansaral** [comment]
<p>I’m experiencing the same issue.</p>

**Comment by mbpolan** [comment, marked as answer]
<p>Same here. Seems like I cannot create new repositories in any of my organizations, and not under my personal account either.</p>
<p>I’m guessing this is now being tracked:</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://dka575ofm4ao0.cloudfront.net/pages-favicon_logos/original/36420/akacZEQQfOBdc7ftyxJt" class="site-icon" width="32" height="32">
      <a href="https://www.githubstatus.com/incidents/n6v2xymbsjts" target="_blank" rel="noopener nofollow ugc">githubstatus.com</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/1791287/181066008-76f794c4-1c34-41ea-9e5a-54fc77707a47.png" class="thumbnail onebox-avatar" width="120" height="120">

<h3><a href="https://www.githubstatus.com/incidents/n6v2xymbsjts" target="_blank" rel="noopener nofollow ugc">Incident with API Requests</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by canuckjacq** [comment]
<p>Welcome :wave: <a class="mention" href="/mbpolan">@mbpolan</a> <a class="mention" href="/welovelain">@welovelain</a> <a class="mention" href="/hasancansaral">@hasancansaral</a></p>
<p>Is anyone still having trouble with this?</p>

**Comment by welovelain** [comment]
<p>Mine got fixed yesterday</p>

**Comment by hasancansaral** [comment]
<p><a class="mention" href="/canuckjacq">@canuckjacq</a> Mine is fixed as well. Thanks.</p>

**Reply by Strix323** [reply]
How can I fix mine? I'm facing the same issues
